components/mpas-ocean/src/shared/mpas_ocn_vmix_cvmix.F

```diff
@@ -65,7 +65,7 @@ module ocn_vmix_cvmix
    type(cvmix_tidal_params_type)  :: cvmix_tidal_params
 
    logical :: cvmixOn, cvmixConvectionOn, cvmixKPPOn
-   real (kind=RKIND) :: backgroundVisc, backgroundDiff
+   real (kind=RKIND) :: multVal, backgroundVisc, backgroundDiff
```

Comment:
@vanroekel Searching for this value doesn't bring up anything for me. Where is it assigned?